values.yaml

```diff
@@ -14,7 +14,7 @@ kobotoolbox:
   # - KPI Cache - 4
   # - Kobocat Cache - 5
   # - Service account - 6
-  redis: "redis://:pass@redis-master:6379"  # Do not add / nor /1 suffix
+  #redis: "redis://:pass@redis-master:6379"  # Do not add / nor /1 suffix
   #redisSession: "redis://:pass@redis-master:6379/1"
   #serviceAccountBackend: "redis://:pass@redis-master:6379/1"
```

Comment:
Not from this PR, but I think if we want to keep this as a template, we should be consistent with numbers just above and use 6. `#serviceAccountBackend: "redis://:pass@redis-master:6379/6"`